internal/pkg/service/appsproxy/dataapps/api/rule.go

```diff
@@ -14,3 +30,16 @@ type Rule struct {
 	Auth         []ProviderID `json:"auth"`
 	AuthRequired *bool        `json:"authRequired"`
 }
+
+func (r *Rule) RegisterHandler(mux *http.ServeMux, handler http.Handler) error {
+	switch r.Type {
+	case RulePathPrefix:
+		if !strings.HasPrefix(r.Value, "/") {
+			return errors.Errorf(`rule "%s": value "%v" must start with "/"`, r.Type, r.Value)
+		}
```

Comment:
See `mux.Handle`, it may contain `method` or `host`, but we don't support it, right?

Reply:
So the `path` must start with `/`, it is handled in the UI in the same way?

Reply:
It would help me if you could solve it, thanks. For example ask UI team or Miro.